doc.go

```diff
@@ -0,0 +1,58 @@
+/*
+Package fluent is an ORM build around the database/sql package.
+
+Overview
+	- SQL builder
+	- Supported database(s): Postgres
+
+Quickstart
+
+Before you can use the Fluent ORM you have to connect to a database, for example:
+
+	import (
+		"database/sql"
+		_ "github.com/lib/pq"
+		bitbucket.org/sebas_dk/fluent
+
+	)
+	func main() {
+		db, err := sql.Open("postgres", "postgres://pqgotest:password@localhost/pqgotest?sslmode=verify-full")
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		fluent := fluent.New(db)
+		…
+	}
+
+Query Builder
+
+The Fluent ORM uses the tags in the struct to find and set values.
+	type Test struct {
+	    ID int       `sql:"id"`
+	    Name string  `sql:"name"`
+	    Total string `sql:"total"`
+	}
+
+Create Record
+  test := Test{Name: "user_1", Total: 12.00}
+  fluent.Table("test").Insert(test)
```

Comment:
The following would be more fluent I guess.

`record := Test{Name: "user_1", Total: 12.00}`
`fluent.Table("test").Insert(record)`